generator/generate_service.go

```diff
@@ -1728,8 +1732,8 @@ func (g *generateCmd) generateRun() (*PartialGenerator, error) {
 		),
 		),
 		jen.Defer().Qual(
-			"github.com/lightstep/lightstep-tracer-go", "FlushLightStepTracer",
-		).Call(jen.Id("tracer")),
+			"github.com/lightstep/lightstep-tracer-go", "Flush",
+		).Call(jen.Id("context.Background()"), jen.Id("tracer")),
```

Comment:
You need to use Qual here context is missing in the imports.

Reply:
Hi, I fix it and update the pr

Reply:
Hi, I fix it and update the pr